source/basic.tex

```diff
@@ -3609,16 +3603,16 @@
 with that of ISO/IEC 9899 Programming Language C.}
 
 \pnum
-\indextext{type!incomplete}%
 \indextext{type!incompletely-defined object}%
-\indextext{object type!incompletely-defined}%
 A class that has been declared but not defined, an enumeration type in certain
 contexts~(\ref{dcl.enum}), or an array of unknown
-size or of incomplete element type, is an \defn{incompletely-defined object
-type}.\footnote{The size and layout of an instance of an incompletely-defined
+size or of incomplete element type, is an
+\defnx{incompletely-defined object type}{object type!incompletely-defined}.%
+\footnote{The size and layout of an instance of an incompletely-defined
 object type is unknown.}
-Incompletely-defined object types and \cv\ \tcode{void} are \term{incomplete
-types}~(\ref{basic.fundamental}). Objects shall not be defined to have an
+Incompletely-defined object types and \cv\ \tcode{void} are
```

Comment:
It's not so much that we actively want to spend time on it, but if there's a chance for a drive-by fix, it's usually good to make small improvements. On the other hand, if you have limitless time and would like to do things systematically, even better :-)